amethyst_input/src/scroll_direction.rs

```diff
@@ -0,0 +1,7 @@
+#[derive(Eq, PartialEq, Debug, Copy, Clone, Serialize, Deserialize)]
+pub enum ScrollDirection {
```

Comment:
amethyst::utils::Axis2

Reply:
Sorry I linked the wrong spot. I was saying that you basically reimplemented this thing here: https://github.com/amethyst/amethyst/blob/master/amethyst_core/src/axis.rs

I'd prefer if you would use that instead of having a copy of it under a different name.